<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PNGizer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="widget.qrc">
    <normaloff>:/icon.jpg</normaloff>:/icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(52, 73, 94);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="fileURLEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;3ds&quot;;
border: 5px solid rgb(236, 240, 241);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="chooseBtn">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    border: 5px solid rgb(46, 204, 113);
    background-color: rgb(46, 204, 113);
    border-top-left-radius:7px;
    border-bottom-left-radius:0px;
    border-top-right-radius:0px;
    border-bottom-right-radius:7px;
	font: 14pt &quot;黑体&quot;;
	color: rgb(255, 255, 255);
}
QPushButton:hover{
    border: 5px solid rgb(57, 255, 140);
    background-color: rgb(57, 255, 140);
    border-top-left-radius:7px;
    border-bottom-left-radius:0px;
    border-top-right-radius:0px;
    border-bottom-right-radius:7px;
	font: 14pt &quot;黑体&quot;;
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
    border: 5px solid rgb(39, 174, 96);
    background-color: rgb(39, 174, 96);
    border-top-left-radius:7px;
    border-bottom-left-radius:0px;
    border-top-right-radius:0px;
    border-bottom-right-radius:7px;
	font: 14pt &quot;黑体&quot;;
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>选择文件</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="colorLabel">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border: 5px solid rgb(236, 240, 241);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="colorChooseBtn">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    border: 5px solid rgb(46, 204, 113);
    background-color: rgb(46, 204, 113);
    border-top-left-radius:7px;
    border-bottom-left-radius:0px;
    border-top-right-radius:0px;
    border-bottom-right-radius:7px;
	font: 14pt &quot;黑体&quot;;
	color: rgb(255, 255, 255);
}
QPushButton:hover{
    border: 5px solid rgb(57, 255, 140);
    background-color: rgb(57, 255, 140);
    border-top-left-radius:7px;
    border-bottom-left-radius:0px;
    border-top-right-radius:0px;
    border-bottom-right-radius:7px;
	font: 14pt &quot;黑体&quot;;
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
    border: 5px solid rgb(39, 174, 96);
    background-color: rgb(39, 174, 96);
    border-top-left-radius:7px;
    border-bottom-left-radius:0px;
    border-top-right-radius:0px;
    border-bottom-right-radius:7px;
	font: 14pt &quot;黑体&quot;;
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>选择滤除颜色</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
         </property>
         <property name="title">
          <string>半透明化像素范围</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;3ds&quot;;</string>
              </property>
              <property name="text">
               <string>R</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="rSpin">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;3ds&quot;;
border: 5px solid rgb(236, 240, 241);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;3ds&quot;;</string>
              </property>
              <property name="text">
               <string>G</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="gSpin">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;3ds&quot;;
border: 5px solid rgb(236, 240, 241);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;3ds&quot;;</string>
              </property>
              <property name="text">
               <string>B</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="bSpin">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;3ds&quot;;
border: 5px solid rgb(236, 240, 241);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="applyBtn">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: 5px solid rgb(46, 204, 113);
    background-color: rgb(46, 204, 113);
    border-top-left-radius:7px;
    border-bottom-left-radius:0px;
    border-top-right-radius:0px;
    border-bottom-right-radius:7px;
	font: 14pt &quot;黑体&quot;;
	color: rgb(255, 255, 255);
}
QPushButton:hover{
    border: 5px solid rgb(57, 255, 140);
    background-color: rgb(57, 255, 140);
    border-top-left-radius:7px;
    border-bottom-left-radius:0px;
    border-top-right-radius:0px;
    border-bottom-right-radius:7px;
	font: 14pt &quot;黑体&quot;;
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
    border: 5px solid rgb(39, 174, 96);
    background-color: rgb(39, 174, 96);
    border-top-left-radius:7px;
    border-bottom-left-radius:0px;
    border-top-right-radius:0px;
    border-bottom-right-radius:7px;
	font: 14pt &quot;黑体&quot;;
	color: rgb(255, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>应用</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 5px solid rgb(46, 204, 113);
    background-color: rgb(46, 204, 113);
    border-top-left-radius:7px;
    border-bottom-left-radius:0px;
    border-top-right-radius:0px;
    border-bottom-right-radius:7px;
	font: 14pt &quot;黑体&quot;;
	color: rgb(255, 255, 255);
}
QPushButton:hover{
    border: 5px solid rgb(57, 255, 140);
    background-color: rgb(57, 255, 140);
    border-top-left-radius:7px;
    border-bottom-left-radius:0px;
    border-top-right-radius:0px;
    border-bottom-right-radius:7px;
	font: 14pt &quot;黑体&quot;;
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
    border: 5px solid rgb(39, 174, 96);
    background-color: rgb(39, 174, 96);
    border-top-left-radius:7px;
    border-bottom-left-radius:0px;
    border-top-right-radius:0px;
    border-bottom-right-radius:7px;
	font: 14pt &quot;黑体&quot;;
	color: rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>保存副本</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
       </property>
       <property name="title">
        <string>预览</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="previewLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/bg.jpg);
background-repeat:repeat-xy;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="widget.qrc"/>
 </resources>
 <connections/>
</ui>
